aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,67 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor atleast_1d(const Tensor& self) {
+  switch (self.dim()) {
+    case 0:
+      return self.reshape({1});
+    default:
+      return self;
+  }
+}
+
+std::vector<Tensor> atleast_1d(TensorList tensors) {
```

Comment:
`const TensorList& tensors`?

Reply:
I tried with `const TensorList& tensors` but with that
```
E   ImportError: /home/user/Desktop/Repositories/Pytorch/PytorchENV/lib/python3.7/site-packages/torch/lib/libtorch_cpu.so: undefined symbol: _ZN2at6native10atleast_2dEN3c108ArrayRefINS_6TensorEEE
```

I guess this has to do with how code generation which consumes `native_functions.yaml` declares the signature for it.
Also note the signature for meshgrid
https://github.com/pytorch/pytorch/blob/c20426f86d2b06f93812a38ea513ee3803900ab6/aten/src/ATen/native/TensorShape.cpp#L1521

Reply:
Cool! Thanks for letting me know. I appreciate your trying it.